projects/ng-boosted/src/lib/o-carousel/o-carousel-container/o-carousel-container.component.ts

```diff
@@ -29,13 +29,10 @@ import {
       .icon-Play {
           content: "\eac9";
       }
-      .swiper-pagination-bullets, swiper-container-horizontal {
+      .swiper-pagination-bullets, .swiper-container-horizontal {
         bottom:2.5rem !important;
       }
-      .swiper-button-prev,.swiper-button-next:focus {
```

Comment:
need to preserve focus

Reply:
it has no affect at all, if you want it back i will fix it instead of removing it

Reply:
It has effect if you navigate with tab key

Reply:
my wrong, gonna change it back now

Reply:
done @PigeardSylvain
can you please review again now?